### ISSUE TITLE
Log and plot training vs. validation losses

### ISSUE THREAD

**Issue by sharondevs** [Murthypsty0419/AI-Powered-Accessibility-Assistant#15]
- Generate loss curves to visualize overfitting or underfitting.
